From 6db1a082ded1b88afe7640e6150bd0e54e1a13b8 Mon Sep 17 00:00:00 2001
From: Billy <xiaobing.feng@windriver.com>
Date: Fri, 1 Mar 2013 17:21:25 +0800
Subject: Netfilter: Change xt_qtaguid_ctrl_file group

Fix:
xt_qtaguid permision manager compare uid/gid of xt_qtaguid_ctrl_file
(/proc/net/xt_qtaguid/ctrl)  with processes. Since current uid/gid
is root/root, It only allow processes which uid equal fsuid or fsuid
equal "root" or gid equal "root" to access. For making MediaServer pass
the permision check, change xt_qtaguid_ctrl_file group to "net_bw_acct",
which make processes in this group pass.

Issue: AXIA-1642
Change-Id: Ic8de8814f84f3e6e55b81fb2d7140edcf110b963
Signed-off-by: Xiaobing Feng <xiaobing.feng@windriver.com>
---
 rootdir/init.rc |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 78c0305..1b8956e 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -130,6 +130,8 @@ loglevel 3
 # Allow everybody to read the xt_qtaguid resource tracking misc dev.
 # This is needed by any process that uses socket tagging.
     chmod 0644 /dev/xt_qtaguid
+# Change group of ctrl, make mediaserver pass xt_qtaguid permision check.
+    chown root net_bw_acct /proc/net/xt_qtaguid/ctrl
 
 on fs
 # mount mtd partitions
-- 
1.7.4.1

